src/js/helpers/LayerFactory.ts

```diff
@@ -244,7 +244,6 @@ export async function LayerFactory(mapView: any, layerConfig: LayerProps): Promi
         title: layerConfig.title,
         visible: layerConfig.visible,
         urlTemplate: layerConfig.url,
-        opacity: layerConfig.opacity.combined,
```

Comment:
@anthonyamaro15 Do we need the if statement below if you are going to add `opacity: layerConfig.opacity.combined` across the board for webtiled layers.

Reply:
Great point!